src/resources/js/move-tickets-dialog.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * Handles the move tickets/move ticket type dialogs.
+ *
+ * @var object tribe_move_tickets
+ * @var object tribe_move_tickets_data
+ */
+
+var tribe_move_tickets = ( 'object' === typeof tribe_move_tickets ) ? tribe_move_tickets : {};
+
+( function( $, obj ) {
+	var $main,
+		$processing,
+		$errors,
+		$current_stage,
+		current_stage_name,
+		$stages,
+		$back,
+		$next,
+		last_direction,
+		skip_choose_event,
+		target_post_id,
+		target_ticket_type_id;
+
+
+	function init() {
```

Comment:
Hmm, well I dig the approach in general (and of course `obj` is in scope for that very reason - to let us make methods public, even if right now none are), but our coding standards state that methods should be private by default. 

This is our first crack at a wizard-style interface and I'm not sure we want to make every method public by default here as that means a lot of work in the future if we want to deprecate stuff and move things around.

So I guess my questions are: 

* Are we deviating from this rule (members should by private by default) in the case of our JS code? (Why?)
* Or are there specific items you'd like to be public (rather than making them all public)?
* I already trigger numerous events and external code is free to utilize those - is _that_ an approach we could extend in preference to making more things public?